sdk/storage/storage-blob/README.md

```diff
@@ -235,27 +232,79 @@ async function streamToString(readableStream) {
     readableStream.on("error", reject);
   });
 }
+```
+
+### Download a blob and convert it to a string (Browsers)
+
+```javascript
+  // Get blob content from position 0 to the end
+  // In browsers, get downloaded data by accessing downloadBlockBlobResponse.blobBody
+  const downloadBlockBlobResponse = await blobClient.download(0);
+  console.log(
+    "Downloaded blob content",
+    await blobToString(downloadBlockBlobResponse.blobBody)
+  );
 
-// An async method returns a Promise object, which is compatible with then().catch() coding style.
-main()
-  .then(() => {
-    console.log("Successfully executed sample.");
-  })
-  .catch((err) => {
-    console.log(err.message);
+// [Browsers only] A helper method used to convert a browser Blob into string.
+export async function blobToString(blob: Blob): Promise<string> {
+  const fileReader = new FileReader();
+  return new Promise<string>((resolve, reject) => {
+    fileReader.onloadend = (ev: any) => {
+      resolve(ev.target!.result);
+    };
+    fileReader.onerror = reject;
+    fileReader.readAsText(blob);
   });
+}
 ```
 
-## More Code Samples
+A complete example of basic scenarios is at [samples/basic.js](https://github.com/azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples/basic.js).
+
+## Troubleshooting
+
+It could help diagnozing issues by turning on the console logging. Here's an example logger implementation. First add a custom logger:
+
+```javascript
+class ConsoleHttpPipelineLogger {
+  constructor(minimumLogLevel) {
+    this.minimumLogLevel = minimumLogLevel;
+  }
+  log(logLevel, message) {
+    const logMessage = `${new Date().toISOString()} ${HttpPipelineLogLevel[logLevel]}: ${message}`;
+    switch (logLevel) {
+      case HttpPipelineLogLevel.ERROR:
+        console.error(logMessage);
+        break;
+      case HttpPipelineLogLevel.WARNING:
+        console.warn(logMessage);
+        break;
+      case HttpPipelineLogLevel.INFO:
+        console.log(logMessage);
+        break;
+    }
+  }
+}
+```
+
+When creating the `BlobServiceClient` instance, pass the logger in the options
+
+```javascript
+  const blobServiceClient = new BlobServiceClient(
+    `https://${account}.blob.core.windows.net`,
+    sharedKeyCredential, {
+      logger: new ConsoleHttpPipelineLogger(HttpPipelineLogLevel.INFO),
+    }
+  );
+```
+
+## Next steps
+
+More code examples
 
 - [Advanced Examples in Wiki](https://github.com/Azure/azure-storage-js/wiki)
```

Comment:
Add a new wiki in the azure-sdk-for-js repo ?

Reply:
Good suggestion. Will log a issue tracking the migration.

---
In reply to: [295984520](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r295984520) [](ancestors = 295984520)

Reply:
Logged #3974

---
In reply to: [296029097](https://github.com/Azure/azure-sdk-for-js/pull/3744#discussion_r296029097) [](ancestors = 296029097,295984520)